src/TextInput.js

```diff
@@ -26,7 +26,8 @@ const sizeVariants = variant({
   }
 })
 
-const TextInput = ({icon, className, block, disabled, ...rest}) => {
+// using forwardRef is important so that other components (ex. SelectMenu) can autofocus the input
+const TextInput = React.forwardRef(({icon, className, block, disabled, ...rest}, ref) => {
```

Comment:
IIRC this is technically a breaking change

Reply:
Yeah looks like it: https://reactjs.org/docs/forwarding-refs.html#note-for-component-library-maintainers